csrgraph/graph.py

```diff
@@ -489,7 +489,7 @@ def read_edgelist(f, directed=True, sep=r"\s+", header=None, **readcsvkwargs):
     Returns : csrgraph
     """
     # Read in csv correctly to each column
-    elist = pd.read_csv(f, sep=sep, header=header, **readcsvkwargs)
+    elist = pd.read_csv(f, sep=sep, header=header, keep_default_na=False, **readcsvkwargs)
```

Comment:
If you're adding a parameter in the `read_csv` here, it should be added in the function parameters at the top level.

Right now, if someone wanted to override this when calling the `read_edgelist` it would be tucked in `readcsvkwargs` and raise a double argument value error in this line.

Reply:
added a default value at the top level, feel free to merge

you may reproduce this issue on the main branch by loading a csv with string values like "null" (which is what happened to me)

Reply:
Thanks, I'll merge it and add a unittest to enforce the behavior afterwards.